### PR TITLE
Address John Scudder comments

### LINKED ISSUE #157: John Scudder's ballot comments
> 1. In the Introduction, “a continuous media” makes me sad because of the
> disagreement in number. I get that this may reflect common usage but it still
> made me wince. The obvious way to make it conventionally grammatical would be
> “a continuous media stream” but I suppose that would make the definition
> recursive. :-( So, I don’t have a good solution to offer but maybe you do; if
> so that’d be nice.
> 
> 2. Introduction, “match to client's consumption rate” should be “match the
> client's consumption rate”.
> 
> 3. In Section 3.6, “Bittorrent favored peers
>    who uploaded as much as they downloaded, so that new Bittorrent users
>    had an incentive to significantly increase their upstream bandwidth
>    utilization.”
> 
> I think you don’t mean “so that”, but just “so” — the implied causal arrow is
> reversed by the “that”, you mean something like “therefore”, right?
> 
> 4. In 3.7 you seem to have a stray close brace? “}”
> 
> 5. In 3.7 you misspelled Craig Labovitz’s name as “Labowitz”. The "v" spelling
> is correct.
> 
> 6. Section 5.5.3, s/detction/detection/
> 
> 7. Section 6.1, s/tansport/transport/
> 
> 8. Section 6.1, “approaches like the above generally
>    experiences”, should be “generally experience” (agreement in number).
> 
> 9. Section 7, “prevent media steam manipulation” should be “stream”.
> 
> 10. Section 7.2 has
> 
>    The choice of whether to involve intermediaries sometimes requires
>    careful consideration.  As an example, when ABR manifests were
>    commonly sent unencrypted some networks would modify manifests during
>    peak hours by removing high-bitrate renditions in order to prevent
>    players from choosing those renditions, thus reducing the overall
>    bandwidth consumed for delivering these media streams and thereby
>    improving the network load and the user experience for their
>    customers.  Now that ubiquitous encryption typically prevents this
>    kind of modification, in order to maintain the same level of network
>    health and user experience across networks whose users would have
>    benefitted from this intervention a media streaming operator
>    sometimes needs to choose between adding intermediaries who are
>    authorized to change the manifests or adding significant extra
>    complexity to their service.
> 
> It wasn’t immediately obvious to me as a reader what recourse (other than
> intermediaries) the media streaming operator would have, regardless of their
> willingness to add extra complexity.
> 
> - Are you implicitly referencing something you’ve touched on elsewhere in the
> document (an xref would be nice if so),
> 
> - or are such “extra complexity” solutions known and published (again, a ref
> would be nice),
> 
> - or known but proprietary special sauce (maybe there’s nothing to be done,
> although a few words indicating this might be suitable),
> 
> - or are you just speculating on the basis that as the saying goes, with enough
> thrust, even a brick can fly?

### PR DESCRIPTION
**Pull request by SpencerDawkins**
Close #157 

@GrumpyOldTroll and @acbegen - note that John's last comment hasn't been addressed in this PR yet.

### CODE REVIEW

**Review by GrumpyOldTroll — APPROVED** [review]
lgtm

File: draft-ietf-mops-streaming-opcons.md [hunk 1]
```diff
@@ -325,7 +325,7 @@ This document examines networking and transport protocol issues as they relate t
 
 This document specifically focuses on streaming applications and defines streaming as follows:
 
-- Streaming is the transmission of continuous media from a server to a client and its simultaneous consumption by the client.
+- Streaming is the continuous transmission of media from a server to a client and its simultaneous consumption by the client.
 
 - Here, "continuous media" refers to media and associated streams such as video, audio, metadata, etc. In this definition, the critical term is "simultaneous," as it is not considered streaming if one downloads a media file and plays it after the download is completed, which would be called download-and-play.
```
Comment (acbegen) [inline draft-ietf-mops-streaming-opcons.md:330]: Here, "media" refers to any type of media and associated streams such as video, audio, metadata, etc.

**Comment by acbegen** [comment]
Added a single comment for the subsequent sentence where you made a change.

File: draft-ietf-mops-streaming-opcons.md (hunk 1 above)
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:330]: I agree - will make this change and merge.

**Pull request merged**